src/styles/interactives.scss

```diff
@@ -78,6 +79,46 @@ div.os-raise-justify-content-end {
   justify-content: end;
 }
 
+select.form-select {
```

Comment:
I have a few related questions which probably also tie to [this comment](https://github.com/openstax/k12-apps-raise/pull/93#issuecomment-1572542128) @MReyna12 made on the PR:

* Are we now overriding Bootstrap's `.form-select` vs extending and defining a custom `.os-form-select`? 
* What are potential unexpected implications of now importing `interactives.scss` in two places? 
    * I believe if we remove the import from `main.scss` the styles in `interactive.scss` would only apply if wrapped within `.os-raise-bootstrap` which might not be what we want :man_shrugging: . 
* I'm wondering about the general pattern we're striving towards here. Previously we were adding classes to `bootstrap.scss` when we wanted to `@extend` an existing BS class (I noticed / commented on [this recent exception](https://github.com/openstax/k12-apps-raise/pull/89/files#r1214621226)). There is now a coupling between the two files :thinking: .
    * :point_up: is really getting at the helping our "future selves" point raised by @MReyna12 :joy: . 
* I left a related class naming convention comment in the [culminating PR](https://github.com/openstax/k12-apps-raise/pull/89/files#r1214638694).

Reply:
I've addressed each bullet in order below:

1. A number of our html elements have at least the `.os-raise-bootstrap` class attached to it plus any additional bootstrap classes we add. For example, the `select` tag has `.os-raise-bootstrap` plus `.form-select` (because `.form-select` is a bootstrap class). This means that we are working with a higher initial CSS specificity for elements that have a bootstrap class attached to it. Importing `interactives.scss` into `bootstrap.scss` allowed my `select.form-select` class to apply its styles over the default bootstrap `.form-select` class (though if for some reason styles in `select.form-select` didn't apply then the default `.form-select` styles would then go into effect). Without importing `interactives.scss` into `bootstrap.scss` means that we need to move (I think) all of the select related styling into `bootstrap.scss` using the typical extend logic.
2. You are correct in your thinking. Unsure on the unexpected implications - when I added the import I messed around with the MC / MS styles and the dropdown styles and nothing stood out to me.
3. I am 100% fine with the previous pattern of adding classes to `bootstrap.scss` when we want to `@extend` an existing BS class. A reason I added the import and then flagged was to bring out the point that I can foresee us needing to `@extend` a number of bootstrap classes as we continue to customize the styling of the site and moving away from the default bootstrap styles. At what point are we adding a ton of CSS to `bootstrap.scss` and not in a non-bootstrap stylesheet?
4. I responded to the naming convention point in that PR you linked. I can adjust the names of the classes based on my comments in the aforementioned PR as well as what we decide to do based on the above.